vim/PKGBUILD

```diff
@@ -6,16 +6,17 @@ _topver=9.0
 _patchlevel=2121
 _versiondir="${pkgname}${_topver//./}"
 pkgver=${_topver}.${_patchlevel}
-pkgrel=1
+pkgrel=2
 pkgdesc='Vi Improved, a highly configurable, improved version of the vi text editor'
 arch=('i686' 'x86_64')
 license=('custom:vim')
 url="https://www.vim.org"
 depends=('ncurses' 'libiconv' 'libintl')
 groups=('editors')
 makedepends=('gawk' 'ncurses-devel' 'gcc'
+             'perl-devel' 'python-devel' 'ruby' 'autotools' 'libxcrypt-devel'
              'libiconv-devel' 'gettext-devel') # To satisfy python3/dyn feature #3052
-source=(${pkgname}-${pkgver}.tar.gz::https://github.com/vim/vim/archive/v${pkgver}/${pkgbase}-{pkgver}.tar.gz
+source=(${pkgname}-${pkgver}.tar.gz::https://github.com/vim/vim/archive/v${pkgver}/${pkgbase}-${pkgver}.tar.gz
```

Comment:
How did this work before!?! 😲

Reply:
This was introduced in https://github.com/git-for-windows/MSYS2-packages/pull/140, and I bet that at some stage I was so fed up with the entire mess that is GitHub Actions' lack of debugging support that I just used `action-tmate` and did it manually.